modules/packages/LinearAlgebra.chpl

```diff
@@ -381,7 +381,7 @@ proc Matrix(const Arrays: ?t  ...?n) where isArrayType(t) && t.rank == 1 {
 */
 proc Matrix(const Arrays: ?t ...?n, type eltType) where isArrayType(t) && t.rank == 1 {
   // TODO -- assert all array domains are same length
-  //         Can this be done via type query?
+  //         using type query (when language allows)
```

Comment:
I think we can remove this TODO comment. I am not sure if this is something the language should or will allow eventually.

Reply:
That makes sense. It should be removed now.